uni/lib/controller/load_info.dart

```diff
@@ -51,13 +51,15 @@ Future loadRemoteUserInfoToState(Store<AppState> store) async {
       trips = Completer(),
       lastUpdate = Completer(),
       restaurants = Completer(),
+      libraryOccupation = Completer(),
       calendar = Completer();
 
   store.dispatch(getUserInfo(userInfo));
   store.dispatch(getUserPrintBalance(printBalance));
   store.dispatch(getUserFees(fees));
   store.dispatch(getUserBusTrips(trips));
   store.dispatch(getRestaurantsFromFetcher(restaurants));
+  store.dispatch(getLibraryOccupationFromFetcher(libraryOccupation));
```

Comment:
Besides this, since you're saving the info in the database, you should update `loadLocalUserInfoToState` accordingly.